frontend/ios/fastlane/README.md

```diff
@@ -15,6 +15,22 @@ For _fastlane_ installation instructions, see [Installing _fastlane_](https://do
 
 ## iOS
 
+### ios build
+
+```sh
+[bundle exec] fastlane ios build
```

Comment:
aren't there some more additional params needed like `buildConfig, flavor, version, etc`

Reply:
Yes they are, but the parameters are not added to the README.